### PR TITLE
bb: Delete stale files

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

File: pkg/uroot/builder/bb/bb.go [hunk 1]
```diff
@@ -259,6 +259,11 @@ func RewritePackage(env golang.Environ, pkgPath, bbImportPath string, importer t
 		return err
 	}
 	dest := filepath.Join(buildp.Dir, ".bb")
+	// If .bb directory already exists, delete it. This will prevent stale
+	// files from being included in the build.
+	if err := os.RemoveAll(dest); err != nil {
+		return err
```
Comment (hugelgupf) [inline pkg/uroot/builder/bb/bb.go]: Can you add an error message here?

Comment (rjoleary) [reply pkg/uroot/builder/bb/bb.go]: Done!

File: pkg/uroot/builder/bb/bb.go [hunk 2]
```diff
@@ -259,6 +259,11 @@ func RewritePackage(env golang.Environ, pkgPath, bbImportPath string, importer t
 		return err
 	}
 	dest := filepath.Join(buildp.Dir, ".bb")
+	// If .bb directory already exists, delete it. This will prevent stale
```
Comment (rminnich) [inline pkg/uroot/builder/bb/bb.go]: should we instead just drop this into a os.TempDIr somewhere? 

removeall scares me. Anyway, I'll approve it, nice catch if this is what it was.

Comment (hugelgupf) [reply pkg/uroot/builder/bb/bb.go]: We can't. I wish.

Go build system reasons (vendor dependencies)

**Pull request merged**